core/src/main/scala/cats/data/Validated.scala

```diff
@@ -265,6 +265,30 @@ sealed abstract class Validated[+E, +A] extends Product with Serializable {
       case Valid(a)   => Right(a)
     }
 
+  /**
+   * Converts the value to an EitherT[F, E, A]
+   *
+   * Example:
+   * {{{
+   * scala> import cats.syntax.all._
+   * scala> given ExecutionContext = ExecutionContext.global
+   *
+   * scala> val v1 = "error".invalid[Int]
+   * scala> val v2 = 123.valid[String]
+   *
+   * scala> v1.toEitherT[Future].value
+   * res0: Future[Either[String, Int]] = Future(Left(error))
+   *
+   * scala> v2.toEitherT[Future].value
+   * res1: Future[Either[String, Int]] = Future(Right(123))
+   * }}}
+   */
+  def toEitherT[F[_]: Applicative]: EitherT[F, E, A] =
+    this match {
+      case Invalid(e) => EitherT.leftT(e)
+      case Valid(a)   => EitherT.rightT(a)
+    }
```

Comment:
And perhaps

```suggestion
    EitherT.fromEither(this.toEither)
```

is cleaner